solutions/swb-reference/integration-tests/tests/isolated/environmentTypeConfigs/create.test.ts

```diff
@@ -0,0 +1,166 @@
+    try {
+      await adminSession.resources.environmentTypes.environmentType(envTypeId).configurations().create(
+        {
+          name: 'name',
+          description: 'description',
+          params: []
+        },
+        false
+      );
+    } catch (e) {
+      checkHttpError(
+        e,
+        new HttpError(400, {
+          statusCode: 400,
+          error: 'Bad Request',
+          message: 'type: Required'
+        })
+      );
+    }
+  });
+
+  test('fails when trying to create without params', async () => {
+    try {
+      await adminSession.resources.environmentTypes.environmentType(envTypeId).configurations().create(
+        {
+          name: 'name',
+          description: 'description',
+          type: 'type'
+        },
+        false
+      );
+    } catch (e) {
+      checkHttpError(
+        e,
+        new HttpError(400, {
+          statusCode: 400,
+          error: 'Bad Request',
+          message: 'params: Required'
+        })
+      );
+    }
+  });
+
+  test('fails when trying to create with invalid prop', async () => {
+    try {
+      await adminSession.resources.environmentTypes.environmentType(envTypeId).configurations().create(
+        {
+          type: 'typeTest',
+          description: 'description',
+          name: 'name',
+          invalidProp: 'invalidValue',
+          params: []
+        },
+        false
+      );
+    } catch (e) {
+      checkHttpError(
+        e,
+        new HttpError(400, {
+          statusCode: 400,
+          error: 'Bad Request',
+          message: ": Unrecognized key(s) in object: 'invalidProp'"
+        })
+      );
+    }
+  });
+
+  test('fails when trying to create with invalid environment Type Id', async () => {
+    try {
+      await adminSession.resources.environmentTypes
+        .environmentType('et-prod-1234567890124,pa-1234567890124')
+        .configurations()
+        .create(
+          {
+            type: 'typeTest',
+            description: 'description',
+            name: 'name',
+            params: []
+          },
+          false
+        );
+    } catch (e) {
+      checkHttpError(
+        e,
+        new HttpError(400, {
+          statusCode: 400,
+          error: 'Bad Request',
+          message: `Could not create environment type config because environment type et-prod-1234567890124,pa-1234567890124 does not exist`
+        })
+      );
+    }
+  });
+
+  test('fails when trying to create with invalid environment Type id format', async () => {
+    try {
+      await adminSession.resources.environmentTypes
+        .environmentType('wrong-foramt-env-type-id')
+        .configurations()
+        .create(
+          {
+            type: 'typeTest',
+            description: 'description',
+            name: 'name',
+            params: []
+          },
+          false
+        );
+    } catch (e) {
+      checkHttpError(
+        e,
+        new HttpError(403, {
```

Comment:
This returns a 403? I feel like it should be a 400. Usually this is what gets returned when a route isn't in the static auth file.

Reply:
this ends up being a bad request as adding an `id` with wrong format the URL will not match anymore as it looks for the regex, adding a valid non existing  `id` will return a 400

Reply:
oh yaaaa i forgot all about that bug :-/